Apps/Sandcastle/gallery/3D Tiles Batch Table Hierarchy.html

```diff
@@ -89,9 +89,9 @@
 }));
 
 tileset.readyPromise.then(function() {
-    var boundingSphere = tileset.boundingSphere;
-    viewer.camera.viewBoundingSphere(boundingSphere, new Cesium.HeadingPitchRange(0.0, -0.3, 0.0));
-    viewer.camera.lookAtTransform(Cesium.Matrix4.IDENTITY);
+    viewer.zoomToTileset(tileset, new Cesium.HeadingPitchRange(0.0, -0.3, 0.0));
```

Comment:
This one is not based on the bounding sphere, so we don't need to wait for the readyPromise.